examples/bri-3/src/bri/zkp/api/dtos/request/createProof.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsNotEmpty } from 'class-validator';
+import DocumentObject from '../../../types/document';
+
+// TODO: Revisit validations rules once we move into business logic implementation
+export class CreateProofDto {
+  @IsNotEmpty()
```

Comment:
referencing the createProof.dto.spec.ts ~ if this field is labelled @IsNotEmpty should this error should be expected somewhere in the spec.